roofit/roofit/src/RooFitComputeInterface.h

```diff
@@ -0,0 +1,22 @@
+#ifndef ROOFITCOMPUTEINTERFACE_H
+#define ROOFITCOMPUTEINTERFACE_H
+
+#include "RooSpan.h"
+
+class RooAbsReal;
+class RooListProxy;
+namespace BatchHelpers {
+  struct RunContext;
+  class BracketAdapterWithMask;
+}
+
+namespace RooFitCompute {
+  class RooFitComputeInterface {
+  public:
+    virtual ~RooFitComputeInterface() {}
+  };
+
+  extern RooFitComputeInterface* dispatch;
+}
```

Comment:
Some short comments here would be nice. Since there's no `.cxx` for this, they have to be directly above the class, so doxygen picks them up.
What should be in the comments:
What is this class? What do we use it for?

For the extern pointer:
Where is this pointer? What do we use it for?

Reply:
Comments added.